impexp-config/src/main/resources/org/citydb/config/i18n/language_de.properties

```diff
@@ -231,6 +231,8 @@ kmlExport.dialog.error.collada2gltf.notExists=<html><body width="300px">Das COLL
 kmlExport.dialog.error.collada2gltf.notExecutable=<html><body width="300px">Das COLLADA2glTF-Tool {0} kann nicht ausgef�hrt werden.<br><br>Stellen Sie sicher, dass ausreichende Ausf�hrungsberechtigungen bestehen.</body></html>
 kmlExport.dialog.error.elevation=Google Elevation API Fehler
 kmlExport.dialog.error.elevation.apiKey=<html>Der Google Elevation API ben�tigt einen API Key.<br><br><b>Hinweis:</b> Bitte geben Sie einen API Key in den allgemeinen Voreinstellungen ein<br>oder �ndern Sie die Export-Einstellungen.</html>
+kmlExport.dialog.warn.ade.unsupported=<html><body width="300px">Die nachfolgenden CityGML ADEs werden von diesem VIS-Exporter nicht unterst�tzt:<br><br>{0}<br><br><b>Hinweis:</b> Bitte pr�fen \
```

Comment:
I think we are using "VIS Export" without a hyphen at other places.

Reply:
updated here 6b6ad74